create_coda_files/create_coda_files.py

```diff
@@ -58,32 +64,48 @@ def _open_scheme(filepath):
     traced_json_output_path = args.traced_json_output_path
     coda_output_path = args.coda_output_path
     coding_scheme_path = args.coding_scheme_path
-    if args.auto_cleaner not in AUTO_CLEANERS.keys():
-        auto_cleaner = args.auto_cleaner
-        auto_cleaner = AUTO_CLEANERS[auto_cleaner]
+    if args.has_yes_no == 'True':
+        has_yes_no = True
+    elif args.has_yes_no == 'False':
+        has_yes_no = False
+    else:
+        raise Exception('<has-yes-no> only takes the strings "True" or "False"')
+    if args.auto_cleaner in AUTO_CLEANERS.keys():
+        auto_cleaner = AUTO_CLEANERS[args.auto_cleaner]
     else:
+        print('There is no matching cleaner for: {}'.format(args.auto_cleaner))
         auto_cleaner = None
 
     # Load in the coding scheme
-    code_scheme = _open_scheme(coding_scheme_path)
+    code_scheme = open_scheme(coding_scheme_path)
+
 
     # Load traced data list from JSON file
     with open(json_input_path, 'r') as f:
         list_td = TracedDataJsonIO.import_json_to_traced_data_iterable(f)
 
     # Filter out test messages sent by AVF.
-    list_td = [td for td in list_td
-    if not td.get('test_run', False)]
+    list_td = [td for td in list_td 
+                if not td.get('test_run', False)]
```

Comment:
You're overindented by a space.
I would put this on one line.

Reply:
fixed